### PR TITLE
Dont increment build number for PRs

### PR DESCRIPTION
**Pull request by RobertBColton**
Ok, so apparently when you have a YAML for AppVeyor, it overrides all settings you put in the UI that are not UI only. This means that just by having a YAML, even if it's empty, you won't be able to configure certain options from the UI. One of these options is one that stops pull requests from incrementing the build number.

I asked AppVeyor: https://github.com/appveyor/ci/issues/2037#issuecomment-358083139

So basically this option just has to go in our YAML and then the build number will only increase when it does a merge build on master.

### CODE REVIEW

**Comment by CyberShadow** [comment]
If you have an `appveyor.yml`, you have to put all settings in `appveyor.yml` (with the exception of some security-sensitive ones that *must* be done via the website).

**Comment by RobertBColton** [comment]
Hrm I didn't think about it that way, however, how can that be when enabling "Rolling Builds" from the UI seems to work? As well as the option to not "cancel running builds"? Those don't seem like security-sensitive options to me necessarily...
